### ISSUE TITLE
merge back `rxode2`

### ISSUE THREAD

**Issue by mattfidler** [nlmixr2/rxode2#687]
CRAN:  

Consider merging `rxode2`, though they originally wanted them split

> One comment for the last release flood: if a new rxode2parse always  breaks most of

>   babelmixr2, nlmixr2, nlmixr2est, nlmixr2extra, nlmixr2plot, nonmem2rx,   rxode2, rxode2et

> then these should really be merged ...

**Comment by billdenney** [comment]
Sigh

**Comment by mattfidler** [comment]
Indeed.  I asked for an exception on compile time, and they haven't gotten back to me yet...

My guess is they will not ant they will simply not accept any more releases of `rxode2parse` unless it is merged in some way.
